src/main/java/com/manardenza/console/ListMenu.java

```diff
@@ -0,0 +1,62 @@
+package com.manardenza.console;
+
+import lombok.Getter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ListMenu {
+
+    @Getter
+    private static List<String> siteHeader = new ArrayList<>();
+    @Getter
+    private static List<String> authorizationHeader = new ArrayList<>();
+    @Getter
+    private static List<String> authorizationMenu = new ArrayList<>();
+    @Getter
+    private static List<String> serviceHeader = new ArrayList<>();
+    @Getter
+    private static List<String> serviceMenu = new ArrayList<>();
+    @Getter
+    private static List<String> bookingHeader = new ArrayList<>();
+    @Getter
+    private static List<String> bookingMenu = new ArrayList<>();
+    @Getter
+    private static List<String> findRoomHeader = new ArrayList<>();
+    @Getter
+    private static List<String> findRoomMenu = new ArrayList<>();
+
+    private ListMenu() {
+    }
+
+    public static void makeMenus() {
+        siteHeader.add("**************************************************************");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "**************************************************************" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)